storage.conf

```diff
@@ -46,6 +46,14 @@ additionalimagestores = [
 # remap-user = "storage"
 # remap-group = "storage"
 
+# Root-auto-userns-user is a user name which can be used to look up one or more UID/GID
+# ranges in the /etc/subuid and /etc/subgid file.  These ranges will be partioned
+# to containers configured to create automatically an user namespace.  Containers
+# configured to automatically create an user namespace can still overlap with containers
+# having an explicit mapping set.
+# This setting is ignored when running as rootless.
+# root-auto-userns-user = "storage"
```

Comment:
I would prefer default should be "containers",

Reply:
:+1: on "containers"